src/de/cuuky/varo/game/world/setup/AutoSetup.java

```diff
@@ -100,10 +98,12 @@ private void setupLobby() {
                 return;
             }
 
-            if (!file.exists())
-                new LobbyGenerator(lobby, ConfigSetting.AUTOSETUP_LOBBY_GEN_HEIGHT.getValueAsInt(), ConfigSetting.AUTOSETUP_LOBBY_GEN_SIZE.getValueAsInt());
-            else
-                new LobbyGenerator(lobby, file);
+            File schematicFile = new File(ConfigSetting.AUTOSETUP_LOBBY_SCHEMATIC_FILE.getValueAsString());
+            if (ConfigSetting.AUTOSETUP_LOBBY_SCHEMATIC_ENABLED.getValueAsBoolean() && schematicFile.exists()) {
```

Comment:
There should probably be some kind of warning/error if `AUTOSETUP_LOBBY_SCHEMATIC_ENABLED` is `true` but the file doesn't exist.